src/audio/midi.rs

```diff
@@ -36,7 +36,7 @@ pub fn latest_control_value(
 ) -> Option<ControlValue> {
     messages.iter().rev().find_map(|message| match message {
         MidiMessage::ControlChange(ch, function, value)
-            if ch == &*slider.channel && function == &*slider.control_change =>
+            if ch == &slider.channel && function == &slider.control_change =>
```

Comment:
Ok in https://github.com/peterstuart/pedals/pull/23#discussion_r637418782, you removed the `&`'s but here are removing the `*`'s

I'm following the concept of the removal of the wrapper types that were unnecessarily there just for deserialization purposes, but not clear on the relationship of this branch to `tap-tempo` or the resulting merge conflicts

Reply:
Ok, this is cleaned up now and niether `&` nor `*` are necessary.